google/oauth2/reauth.py

```diff
@@ -292,6 +293,8 @@ def refresh_grant(
             token has a wild card scope (e.g.
             'https://www.googleapis.com/auth/any-api').
         rapt_token (Optional(str)): The rapt token for reauth.
+        enable_reauth_refresh (Optional[bool]): Whether reauth refresh flow
+            should be used. The default value is False.
```

Comment:
Maybe add something about the fact that if used, the caller should update ADC credentials with the new rapt token to avoid constant re-triggering.

Reply:
added a comment saying this method for gcloud to use only